.hooks/README.md

```diff
@@ -0,0 +1,21 @@
+# Git Hooks
+
+Run conventional commit checks locally without any tool installation
+
+## Usage
+
+Adjust the git commit hook configuration as follows, from the root directory of pcap-release:
+
+```shell
+git config core.hooksPath .hooks
+```
+
+Please note that any hooks you had in other directories may not work.
+
+Alternatively you can also soft-link the commit-msg script:
+
+```shell
+ln -s .hooks/commit-msg ./git/hooks/commit.msg
```

Comment:
Did you try this? Looks like a typo to me:
```suggestion
ln -s .hooks/commit-msg .git/hooks/commit-msg
```

And here the inverse applies: If you have hooks configured elsewhere this won't work out of the box.

Reply:
seems like I haven't 😅
thanks. Will apply it as mixup.